frontend/src/auth/AuthenticationContext.js

```diff
@@ -0,0 +1,5 @@
+import { useContext, createContext } from 'react'
```

Comment:
This file is a duplicate of `AuthContext` can be deleted, because it is not used.